docs/user/manage_facility.rst

```diff
@@ -0,0 +1,15 @@
+.. _manage_facility:
+
+Manage Facility
+~~~~~~~~~~~~~~~~
+
+You can edit facility configuration settings in Kolibri from the **Facility** tab in your **Manage** dashboard.
+
+#. Activate the options you want to make available for the users of your facility.
+#. Click **Save changes** to apply and finish.
+
+.. image:: img/manage_facility.png
```

Comment:
There are currently fewer settings in Kolibri than what is showing in this image

![screen shot 2017-05-10 at 1 11 40 pm](https://cloud.githubusercontent.com/assets/6668144/25919086/63ed9258-3582-11e7-9227-619a4af5fc99.png)